layers/+distribution/spacemacs-base/packages.el

```diff
@@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
         i.e. `SPC m h h`, to lookup the source of the definition,
         while falling back to `evil-lookup'"
         (interactive)
-        (condition-case nil
+        (condition-case-unless-debug nil
```

Comment:
My perspective is that you enable debugging because you want a backtrace and I'd just assume always generate one. 

If you're sure that someone would never want the backtrace here I'm ok removing it.